specification/2.0/appendix/performance-metrics/README.md

```diff
@@ -0,0 +1,77 @@
+|[vertexcount]    | Complexity  |Scene     |Total number of vertices used by a model in a scene             |  
+|[nodecount]      | Complexity  |Scene     |Max nodecount in scene (add upp all nodes in a scene)           |  
+|[primitivecount] | Complexity  |Scene     |Total number of referenced primitives (per scene).  This figure is the un-batched number of primitives, engines may optimize if primitives and meshes share textures. |  
+|[textures]       | Complexity  |Scene     |Flags specifying presence of materials and it's texture usage, this is the aggregated max usage. BASECOLOR, METALLICROUGHNESS, NORMAL, OCCLUSION, EMISSIVE, (SPECULARGLOSS) |  
+|[attributes]     | Memory      | Asset    | The total memory footprint, in bytes, for the models attribute buffer storage |  
+|[texturesizes]  | Memory      | Asset    | The sizes of defined textures in the Asset. |  
+
+
+# Scene #  
+
+Metrics that describe properties of a scene.  
+glTF models may contain multiple scenes, only one scene will be visible at a time.  
+To accomodate this the following metrics are calculated on a per-scene basis.  
+It is possible to reference different meshes (nodes) using different scenes, this makes it possible to have the same texture assets but reference different primitives that use alternative materials (textures).  
+Ie one model could have 3 different scenes, where the same accessors (position, uv) are used but different primitives that reference materials with varying number of texture channels.  
+
+[nodecount]  
+This represent the number of traversed nodes in a scene.  
+This is calculated by traversing nodes in each scene (depth or breadth first does not matter) - for each node increase the nodecount.  
+
+[verticecount]  
+This represents the number of drawn vertices for each scene in the model.  
+This is calculated by traversing the nodes in each scene.  
+For each mesh use the POSITION attribute in each primitive, get the Accessor and add up the count field.  
+
+[primitivecount]  
+This represents the number of primitives for each scene  
+Each primitive references a material with 0 or more texture sources, has attributes and accessors.  
+
+
+[textures]  
+This value represents the texture sources that are used by a scene.  
+This is the max value from the most 'complex' primitive that will be referenced by a Node in the scene.  
+The goal of this metric is to provide a worst case texture usage where texturecount and complexity is known.  
+Ie it is known if a texture is BASECOLOR or if part of PBR such as METALICROUGHNESS and now many texture sources that are used.  
+
+[attributes]  
+This value represent the total attribute buffer usage of the model. This value is calculated by adding up the size, in bytes, of all buffer objects in the Asset.  
+This may give an indication of the runtime memory footprint of the buffers needed for the model.  
+
+[texturesizes]
```

Comment:
This information is relevant for two aspects: On the one hand, the memory that will be occupied by the texture. But also whether the client can support the texture at all (i.e. whether the size is not larger than what `glGet(GL_MAX_TEXTURE_SIZE)` returns. 

In any case: The information in the example is given as a *string*, like `"128 * 100"`, so it has to be parsed in some way. People will have to rely on the structure for that - namely, the string *MUST* always have the format `<integer> * <integer>` - this could be worth mentioning. 

In some ways, it might be easier/cleaner to specify the sizes using two integer values, like 

    "textureSizes" : [
        [ 1024, 512 ],
        [ 512, 512 ],
        [ 128, 256 ],
     ],

or so, but that's just a thought...

Reply:
Added something similar by adding Dimension that is Integer[2] containing width and height.